build_tools/ci/cpu_comparison/run.py

```diff
@@ -61,28 +85,50 @@ class BaseTest(ABC):
 
     def __init__(
         self,
-        run_on_target=["npu1_4col"],
-        aie_compilation_flags=None,
-        use_chess=False,
+        name="",
+        running_params: RunningParams = RunningParams(),
```

Comment:
Here, and everywhere else below, use `None` as the default value instead to avoid potential mutability issues: https://www.geeksforgeeks.org/default-arguments-in-python/. Then inside the function do: 
```
running_params = running_params if running_params is not None else RunningParams()
```